backend/api/v1/endpoints/research.py

```
@@ -138,7 +140,10 @@ def search_papers(
                 "file_path": f"http://127.0.0.1:8000/uploads/research_papers/{paper.file_path}",
                 "uploader_id": paper.uploader_id,
                 "download_url": f"/papers/download/{paper.id}/",
-                "request_id": f"/request-collaboration/{paper.id}/"
+                "request_id": f"/request-collaboration/{paper.id}/",
+                "original_filename": paper.original_filename
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)